extensions/amazon-dynamodb/runtime/src/main/java/io/quarkus/dynamodb/runtime/SyncHttpClientConfig.java

```diff
@@ -0,0 +1,214 @@
+package io.quarkus.dynamodb.runtime;
+
+import java.net.URI;
+import java.time.Duration;
+import java.util.List;
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class SyncHttpClientConfig {
+
+    /**
+     * Type of the sync HTTP client implementation
+     *
+     * <p>
+     * Accepted values are `URL` or `APACHE`
+     */
+    @ConfigItem(defaultValue = "url")
+    public SyncClientType type;
+
+    /**
+     * The maximum amount of time to establish a connection before timing out.
+     *
+     * <p>
+     * Used by `URL` and `APACHE` clients
+     */
+    @ConfigItem(defaultValue = "2S")
+    public Duration connectionTimeout;
+
+    /**
+     * The amount of time to wait for data to be transferred over an established, open connection before the connection is
+     * timed
+     * out.
+     * 
+     * <p>
+     * Used by `URL` and `APACHE` clients
+     */
+    @ConfigItem(defaultValue = "30S")
+    public Duration socketTimeout;
+
+    /**
+     * Apache HTTP client additional configuration
+     */
+    @ConfigItem(name = ConfigItem.PARENT)
+    public ApacheHttpClientConfig apacheHttp;
```

Comment:
Wondering if it shouldn't be in an `apache` group instead of added to the parent? This way, it's clear it's Apache specific and you don't need any additional documentation.

Reply:
I think it make sense. However, since I wanted to avoid duplications, the connectionTimeout & socketTimeout that are on the parent and apply to URL connection are also applies to Apache. Wondering if it worth to reflect that on the documentation somehow..